diracx-db/src/diracx/db/sql/pilot_agents/schema.py

```diff
@@ -0,0 +1,65 @@
+from sqlalchemy import (
+    DateTime,
+    Double,
+    Index,
+    Integer,
+    String,
+    Text,
+)
+from sqlalchemy.orm import declarative_base
+
+from ..job.schema import EnumBackedBool
+from ..utils import Column, NullColumn
+
+PilotAgentsDBBase = declarative_base()
+
+
+class PilotAgents(PilotAgentsDBBase):
+    __tablename__ = "PilotAgents"
+
+    PilotID = Column("PilotID", Integer, autoincrement=True, primary_key=True)
+    InitialJobID = Column("InitialJobID", Integer, default=0)
+    CurrentJobID = Column("CurrentJobID", Integer, default=0)
+    TaskQueueID = Column("TaskQueueID", Integer, default=0)
+    PilotJobReference = Column("PilotJobReference", String(255), default="Unknown")
+    PilotStamp = Column("PilotStamp", String(32), default="")
+    DestinationSite = Column("DestinationSite", String(128), default="NotAssigned")
+    Queue = Column("Queue", String(128), default="Unknown")
+    GridSite = Column("GridSite", String(128), default="Unknown")
+    Broker = Column("Broker", String(128), default="Unknown")
+    OwnerDN = Column("OwnerDN", String(255))
+    OwnerGroup = Column("OwnerGroup", String(128))
+    GridType = Column("GridType", String(32), default="LCG")
+    GridRequirements = Column("GridRequirements", Text, default="")
+    BenchMark = Column("BenchMark", Double, default=0.0)
+    SubmissionTime = NullColumn("SubmissionTime", DateTime)
+    LastUpdateTime = NullColumn("LastUpdateTime", DateTime)
+    Status = Column("Status", String(32), default="Unknown")
+    StatusReason = Column("StatusReason", String(255), default="Unknown")
+    ParentID = Column("ParentID", Integer, default=0)
+    OutputReady = Column("OutputReady", EnumBackedBool(), default=False)
+    AccountingSent = Column("AccountingSent", EnumBackedBool(), default=False)
```

Comment:
Hi @martynia, I think you copied the DB schema from Dirac `v8.0` here, right?
There have been some simplifications from `v9.0` as you can see here: 

https://github.com/DIRACGrid/DIRAC/blob/826762853ecdd2011bfc9dffa6fef2eeb9842865/src/DIRAC/WorkloadManagementSystem/DB/PilotAgentsDB.sql#L28-L49

Reply:
Thanks for the comment. Fixed.